src/framework/preferences/js/SeparatedPanelPrefsEditor.js

```diff
@@ -232,8 +232,14 @@ var fluid_2_0 = fluid_2_0 || {};
     fluid.prefs.separatedPanel.bindEvents = function (prefsEditor, iframeEnhancer, separatedPanel) {
         // TODO: This binding should be done declaratively - needs ginger world in order to bind onto slidingPanel
         // which is a child of this component
+
```

Comment:
Moved the function, will file the todo as a separate JIRA if it doesn't already exist